contracts/DelegateContribute.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+interface IWVEOLAS {
+    /// @dev Gets the account balance in native token.
+    /// @param account Account address.
+    /// @return balance Account balance.
+    function balanceOf(address account) external view returns (uint256 balance);
+
+    /// @dev Gets the voting power.
+    /// @param account Account address.
+    /// @return balance Account balance.
+    function getVotes(address account) external view returns (uint256 balance);
+}
+
+/// @dev No balance available to delegate.
+/// @param delegator provided delegator address.
+error NoBalance(address delegator);
+
+/// @title DynamicContribution - Ownable smart contract for minting ERC721 tokens
+contract DelegateContribute {
+
+    /// @dev Delegation.
+    /// @param delegator provided delegator.
+    /// @param delegatee provided delegatee.
+    event Delegation(address indexed delegator, address indexed delegatee);
+
+    // wveOLAS address
+    address immutable wveOLAS;
+
+    // Mapping of account address => address
+    mapping(address => address) public mapDelegation;
+
+    // Maps a delegatee to their list of delegators
+    mapping(address => address[]) private delegatorLists;
```

Comment:
with public not needed in getter

Reply:
it didn't work. I tried and it would not work. No idea why but was not possible to access the getter

Reply:
OK, np

Reply:
Public getter only gets one array element by the index, for the full array you need a getter.